acra-censor/handlers/querycapture_handler.go

```diff
@@ -219,8 +221,15 @@ func AppendQueries(queries []*QueryInfo, openedFile *os.File) error {
 
 func SerializeQueries(queries []*QueryInfo) ([]byte, error) {
 	var linesToAppend []byte
+	var tempQueryInfo = &QueryInfo{}
 	for _, queryInfo := range queries {
-		jsonQueryInfo, err := json.Marshal(queryInfo)
+		queryWithHiddenValues, err := RedactSQLQuery(queryInfo.RawQuery)
+		if err != nil {
+			return nil, ErrQuerySyntaxError
```

Comment:
I believe we shouldn't return error if query can't be redacted, just skip it during writing.

I think it's better to skip writing one query to QueryCaptureLog, then not to write all of them due to one error.

Reply:
@vixentael it will be ignored if IgnoreParseError setting will be set in acra-censor config (acra-censor/acra-censor_implementation.go:35, configs/acra-censor.example.yaml:1)

But I think better to add here some log `log.WithError(err).Errorln("Can't hide values")` to understand what and where it happened

Reply:
I think we should add log for sure.

But I'm concerned about the fact that if masking at least one SQL query fails, the code exits from `for` loop, and returns error instead of json lines. I think code should log error, and skip this line (continue parsing other queries).